src/serial_bridge_index.cpp

```diff
@@ -913,7 +916,7 @@ std::vector<Utxo> serial_bridge::extract_utxos_from_tx(BridgeTransaction tx, cry
 	}
 
 	std::vector<crypto::key_derivation> additional_derivations;
-	for (const auto &pub : tx.additional_pubs) {
+	for (const auto& pub : tx.additional_pubs) {
```

Comment:
nit: keep the reference symbol on the variable:
```suggestion
	for (const auto &pub : tx.additional_pubs) {
```

Reply:
They're both valid, and they compile fine, as whitespace between auto and & does not affect functionality. However, the commonly preferred style is auto&. The & should be directly attached to auto (auto&)